sympy/integrals/transforms.py

```diff
@@ -1412,14 +1414,60 @@ def inverse_laplace_transform(F, s, t, plane=None, **hints):
     See Also
     ========
 
-    laplace_transform
+    laplace_transform, fast_inverse_laplace
     hankel_transform, inverse_hankel_transform
     """
     if isinstance(F, MatrixBase) and hasattr(F, 'applyfunc'):
         return F.applyfunc(lambda Fij: inverse_laplace_transform(Fij, s, t, plane, **hints))
     return InverseLaplaceTransform(F, s, t, plane).doit(**hints)
 
 
+def fast_inverse_laplace(e, s, t):
```

Comment:
Can you test this? Also, include in the release notes?

Reply:
It should be tested.

It should either be private (with a leading underscore name) or it should have a proper docstring etc. It should only be mentioned in the release notes if it is public.

Reply:
> It should either be private (with a leading underscore name) or it should have a proper docstring etc.

I have made it private since I don't know much about the inner implementation. You can make it a public function and add docs/tests later according to the need.